src/sensorpush_ble/parser.py

```diff
@@ -93,19 +145,22 @@ def _start_update(self, service_info: BluetoothServiceInfo) -> None:
         local_name = service_info.name
         result = {}
         device_type = None
-        for match_name in LOCAL_NAMES:
-            if match_name in local_name:
-                device_type = match_name
-        if (
-            not device_type
-            and "ef090000-11d6-42ba-93b8-9dd7ec090ab0" in service_info.service_uuids
-        ):
-            first_manufacturer_data_value_len = len(
-                next(iter(manufacturer_data.values()))
-            )
-            device_type = SENSORPUSH_MANUFACTURER_DATA_LEN.get(
-                first_manufacturer_data_value_len
-            )
+        is_ht1 = False
+
+        if local_name == "s" and SENSORPUSH_UUID in service_info.service_uuids:
+            device_type = "HT1"
+            is_ht1 = True
+        else:
+            for match_name in LOCAL_NAMES:
+                if match_name in local_name:
+                    device_type = match_name
+            if not device_type and (SENSORPUSH_UUID_V2 in service_info.service_uuids):
+                first_manufacturer_data_value_len = len(
+                    next(iter(manufacturer_data.values()))
+                )
+                device_type = SENSORPUSH_MANUFACTURER_DATA_LEN.get(
+                    first_manufacturer_data_value_len
+                )
```

Comment:
This could flow a bit better with returns if you made this a function `determine_device_type`